mindmeld/models/taggers/memm.py

```diff
@@ -154,6 +154,32 @@ def _predict_proba_example(self, example, config, resources):
             seq_log_probs.append([prev_tag, prediction[predicted_tag]])
         return seq_log_probs
 
+    def predict_proba_active_learning(self, examples, config, resources):
```

Comment:
We can update the use of "active_learning" in the function name here as well. If the new function name is say predict_proba_all(), we may want to consider adding this to the base class and implementing it for the other classes. Even if predict_proba_all() simply returns predict_proba() across other subclasses it would make for a more consistent application interface. Ultimately, we may want to refactor such that predict_proba() returns the same thing as predict_proba_all()

Reply:
updated the flag from active_learning to fetch_distribution and predict_proba_active_learning to predict_proba_distribution